lib/bump/cli/commands/deploy.rb

```diff
@@ -5,7 +5,7 @@ class CLI
     module Commands
       class Deploy < Base
         desc "Create a new version"
-        argument :file, required: true, desc: "Path or URL to your API documentation file. Only OpenApi 2.0 (Swagger) specification is currently supported."
+        argument :file, required: true, desc: "Path or URL to your API documentation file. OpenApi 2.0 and 3.0 (Swagger) specifications are currently supported."
```

Comment:
I think that we can fully remove the `Swagger` name, and only talk about OpenAPI. Is that ok for you? (same comment for the 3 changes)

Reply:
👍 

I edited the initial commit